.github/workflows/test.yml

```
@@ -0,0 +1,62 @@
+
+jobs:
+  websocket-server-checks:
+    name: Websocket Server Checks
+    runs-on: ubuntu-latest
+    
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x]
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)